docs/index.rst

```diff
@@ -27,6 +27,12 @@ The configuration options supported by the scale filter are detailed in the tabl
 |                 | If blank, the filter will be applied to all assets.              |
 |                 | This is useful when applying the filter in the north.            |
 +-----------------+------------------------------------------------------------------+
+| Datapoint filter| A regular expression to apply to the Datapoint name.                 |
+|                 | If configured, the filter will be applied only to those datapoints   |
+|                 | that match the expression.                                       |
+|                 | If blank, the filter will be applied to all datapoints.              |
+|                 | This is useful when applying the filter in the north.            |
++-----------------+------------------------------------------------------------------+
```

Comment:
There are formatting issues with this addition to the table that causes the entire table to disappear from the web page when rendered. You can see this by [navigating here](https://github.com/fledge-iot/fledge-filter-scale/blob/FOGL-10397/docs/index.rst) and choosing _Preview_.

Reply:
Fixed